### ISSUE TITLE
prisma/zod/* files get all changed after the initial setup

### ISSUE THREAD

**Issue by denik1981** [calcom/cal.com#1791]
Found a bug? Please fill out the sections below. 👍

### Issue Summary

By running `yarn generate-schemas` (post-installation step) all the files within the `prisma/zod` directory changed diffing from HEAD in a single newline missing after the first import.

### Steps to Reproduce

1. `yarn generate-schemas`
2. `git diff --name-only`

```
prisma/zod/attendee.ts
prisma/zod/availability.ts
prisma/zod/booking.ts
prisma/zod/bookingreference.ts
prisma/zod/credential.ts
prisma/zod/dailyeventreference.ts
prisma/zod/attendee.ts
prisma/zod/availability.ts
prisma/zod/booking.ts
prisma/zod/bookingreference.ts
prisma/zod/credential.ts
prisma/zod/dailyeventreference.ts
prisma/zod/destinationcalendar.ts
prisma/zod/eventtype.ts
prisma/zod/eventtypecustominput.ts
prisma/zod/membership.ts
prisma/zod/payment.ts
prisma/zod/remindermail.ts
prisma/zod/resetpasswordrequest.ts
prisma/zod/schedule.ts
prisma/zod/selectedcalendar.ts
prisma/zod/team.ts
prisma/zod/user.ts
prisma/zod/verificationrequest.ts
prisma/zod/webhook.ts
```

Any other relevant information. For example, why do you consider this a bug and what did you expect to happen instead?

### Technical details
- node v14.19.0
- yarn v1.22.17
- git v2.35.1

**Comment by miguelnietoa** [comment]
Yeah, the same happens to me

**Comment by denik1981** [comment]
Don't worry. I have fixed it in a PR (which I'm preparing) but I have to submit the linked issue first.

**Comment by miguelnietoa** [comment]
> Don't worry. I have fixed it in a PR (which I'm preparing) but I have to submit the linked issue first.

Nice! 🚀🚀

**Comment by denik1981** [comment]
Fixed in #1796

**Comment by denik1981** [comment]
This issue is showing up once again but for a different reason. 

I saw that in a recent PR `prisma/zod/*` files have been excluded from formatting (prettierignore) and now there is a mismatch between the committed and formatted version of these files and the ones that are generated by the user in the post-installation step.

Imho these files shouldn't be versioned at all. I don't see the point if they are fully built by `prisma generate`.
Can we add them to `gitignore`?
 
See #1823

**Comment by zomars** [comment]
Solved in #1896 

> Imho these files shouldn't be versioned at all. I don't see the point if they are fully built by prisma generate.
> Can we add them to gitignore?

The idea is to track changes of these files as prisma.schema gets modified. That's the main reason they're prettier ignored.

**Comment by denik1981** [comment]
Thanks for the fix!
